rebar.config

```diff
@@ -7,6 +7,8 @@
    {ibrowse, ".*",
     %% Pin here, becase 555f707 (pr #155) introduces an ipv6 bug we've not fixed
     {git, "https://github.com/cmullaparthi/ibrowse.git", {ref, "c97136cfb61fcc6f39d4e7da47372a64f7fca04e"}}},
+    {lager, ".*",
+        {git, "git://github.com/basho/lager.git", {branch, "master"}}},
```

Comment:
I hesitated adding this, but then I took a look at github and it seems that basically only oc_reporting and chef-server use this library at this point.  We might consider just moving this into the chef-server repository at some point. But for now, I think adding lager is fine as all of our callers already depend on it.